lib/Elastica/Query.php

```diff
@@ -349,9 +349,12 @@ public function addScriptField($name, AbstractScript $script)
      */
     public function addAggregation(AbstractAggregation $agg)
     {
-        if (!array_key_exists('aggs', $this->_params)) {
-            $this->_params['aggs'] = array();
-        }
+        /**
```

Comment:
Another one here.

If this is an issue, aggs should be set in the constructor.